runtime/lua/vim/lsp/tagfunc.lua

```diff
@@ -0,0 +1,69 @@
+local M = {}
+local lsp = vim.lsp
+
+local function mk_tag_item(name, range, uri)
+  local start = range.start
+  return {
+    name = name,
+    filename = vim.uri_to_fname(uri),
+    cmd = string.format(
+      'call cursor(%d, %d)', start.line + 1, start.character + 1
```

Comment:
```suggestion
      'call cursor(%d, %d)|;"', start.line + 1, start.character + 1
```

So seems that the trailing characters thing is not a bug with Nvim, but an oddity of the tags file format.
It's explained in `:h tags-file-format`, in the description for `{tagaddress}` and `{term}`.

Reply:
Sorry, slightly out of my comfort zone with the vim incantations hah. The search pattern looks fine based on my reading, but I have no idea if this is slower/if this causes any issues. cc @nvim-lsp/@gpanders